### PR TITLE
Disable checks

### PR DESCRIPTION
**Pull request by nnikolovGC**
Disables more checks that we do not wish to track

### CODE REVIEW

File: CIS Scripts/1_Set_Organization_Priorities.sh [hunk 1]
```diff
@@ -131,7 +131,7 @@ OrgScore2_4_4="true"
 
 # 2.4.5 Disable Remote Login 
 # SSH
-OrgScore2_4_5="true"
+OrgScore2_4_5="false"
```
Comment (ecoulson-gc) [inline CIS Scripts/1_Set_Organization_Priorities.sh:134]: Is there a reason for this? I.e. remote administration?

Comment (nnikolovGC) [reply CIS Scripts/1_Set_Organization_Priorities.sh:134]: Yep, precisely.

Comment (ecoulson-gc) [reply CIS Scripts/1_Set_Organization_Priorities.sh:134]: Fair enough!

**Review by ecoulson-gc — APPROVED** [review]
Looks good!

**Pull request merged**